nbs/examples/6PERMBU_example.ipynb

```diff
@@ -1,530 +0,0 @@
-{
```

Comment:
I suggest we add the following reference:
- [Taieb, Souhaib Ben and Taylor, James W and Hyndman, Rob J. (2017). Coherent probabilistic forecasts for hierarchical time series. International conference on machine learning ICML.](https://proceedings.mlr.press/v70/taieb17a.html)

And the following paragraph at the beginning of the notebook:
`In this notebook we will explore probabilistic hierarchical sample reconciliation using the PERM-BU method. The PERM-BU (Taieb et. al, 2017) method leverages empirical bottom-level marginal distributions with empirical copula functions (describing bottom-level dependencies) to generate the distribution of the aggregate-level distributions using the bottom-up reconciliation. The sample reordering technique from the PERM-BU method reinjects multivariate dependencies into independent bottom-level samples.`